### ISSUE TITLE
Playback issues in 5.18.0

### ISSUE THREAD

**Issue by anyuta1166** [SimpleMobileTools/Simple-Music-Player#604]
Hello.
After upgrading to version 5.18.0 I have some major issues with playback.

1. Every time after switching to the next track, the player hangs for a few seconds. The track is successfully played, but the the song progress stays at 00:00 and the player doesn't respond to play/pause button.
2. There are a lot of sound interruptions during playback.
3. The player cannot be completely closed. It stays in notification bar forever (until reboot). Pressing "x" (close) button in notification does nothing.

Version 5.17.1 works perfectly without any issues.

Device: Tecno Spark Go 2022 (Android 11).

**Comment by tibbi** [comment]
maybe @naveensingh can take a look at it

**Comment by anyuta1166** [comment]
4. Repeatedly pressing next or previous track button causes the application to completely hang (showing the system dialog "application does not responding" with close and wait buttons).

**Comment by sezanzeb** [comment]
Can't confirm any freezes and such. App runs fast for me with ~7000 lossy files.

Android 13, LineageOS 20-20230604-microG-FP4

**Comment by sezanzeb** [comment]
I get a laggy ui when shuffling through the complete library. Like that it takes some time before it starts playing the next song. Jumping within a track is also really unresponsive in that case.

Works fine when the playlist is shorter.

**Comment by ionuttbara** [comment]
Samsung Galaxy A73, Android 13 + Asus ZenFone ZC520KL Android 8.

**Comment by JillianLaville** [comment]
Found a workaround , on some of these issues with gapless play back disabled.

**Comment by naveensingh** [comment]
@JillianLaville can you tell me exactly what issue was fixed by disabling gapless playback? 

@anyuta1166 can you check if this workaround (disabling gapless playback) works for your second case? (sound interruptions)

**Comment by JillianLaville** [comment]
I had a problem where the reproduction would stop sending audio after a playback, regardless of it being in any kind of repeat mode, it was fixed by moving the progress of the song or time, but after automatically starting a new track it would be muted without the sound of my phone being muted or the sound signal being sent to another devices, it happened regardless of which device I used to listen to, after I disabled the gapless playback, as far as I've seen it will continue playing with sound after a new track plays.

**Comment by naveensingh** [comment]
@JillianLaville are you playing recordings or audio books with silenced parts or is it just casual music?

**Comment by JillianLaville** [comment]
@naveensingh  Just casual Music, shouldn't be any silenced parts, it was the same music I was always listening previously 
 and suddenly had this problem occur consistently a couple days ago.

**Comment by naveensingh** [comment]
Got it. Are you also facing any delays or lags as others have mentioned?

**Comment by JillianLaville** [comment]
Not really, just when it changes track it needs a couple of seconds to actualize the display and title of song, but it doesn't hinder UI traversal or performance.

@naveensingh 
Specs:
Model MotoG42 XT2233-2
Android 12
4GB RAM
2400X1080 Screen
128 ROM

**Comment by naveensingh** [comment]
understood, that will be fixed too.

**Comment by sezanzeb** [comment]
> Just to confirm, the UI lag is observed if you turn on shuffle mode in the player on 5.18.0 and above but not on previous versions?

Not observing lags on 5.17.1

From 5.18 on: Both with and without shuffle. It only happens when playing back every song in my library. It doesn't happen on a smaller subset (like when selecting songs of a particular genre only)

> What device are you using?

Fairphone 4
